backend/src/test/java/quarano/actions/DiaryEventListenerTest.java

```diff
@@ -13,39 +25,40 @@
 import org.mockito.ArgumentCaptor;
 import org.mockito.Captor;
 import org.mockito.Mock;
+import org.springframework.context.support.MessageSourceAccessor;
 import org.springframework.data.util.Streamable;
 
-import quarano.QuaranoUnitTest;
-import quarano.department.TrackedCase;
-import quarano.department.TrackedCaseRepository;
-import quarano.diary.*;
-import quarano.masterdata.Symptom;
-import quarano.tracking.BodyTemperature;
-import quarano.tracking.TrackedPerson.TrackedPersonIdentifier;
-
 /**
  * @author Oliver Drotbohm
  * @author Jens Kutzsche
  */
 @QuaranoUnitTest
 class DiaryEventListenerTest {
 
-	@Mock ActionItemRepository items;
-	@Mock DiaryManagement diaryManagement;
-	@Mock AnomaliesProperties config;
-	@Mock TrackedCaseRepository cases;
+	@Mock
```

Comment:
Can we avoid the reformatting in the core commit?

Reply:
Yes, of course.
Can we avoid this annoying reformatting with the preceding of private?